### PR TITLE
docs(common): fix invite link ASO

### PR DESCRIPTION
**Pull request by fvives99**
### What does this PR do?
Fixed broken link and updated readme.

### Steps to test
1. Click on discord image icon

#### CheckList
- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,9 +4,8 @@ Sitio Web Asociación Blockchain de Costa Rica
...
 
 <div align="center">
-	<a href="https://edenia.com">
-		<img src="https://raw.githubusercontent.com/edenia/.github/master/.github/workflows/images/edenia-logo.png" width="300">
-	</a>
+
+<a href="https://edenia.com"><img width="400" alt="image" src="https://raw.githubusercontent.com/edenia/.github/master/.github/workflows/images/edenia-logo.png"></img></a>
 
 [![Twitter](https://img.shields.io/twitter/follow/EdeniaWeb3?style=for-the-badge)](https://twitter.com/EdeniaWeb3)
 ![Discord](https://img.shields.io/discord/946500573677625344?color=black&label=discord&logo=discord&logoColor=white&style=for-the-badge)
```
Comment (xavier506) [inline README.md:11]: missing link

Comment (fvives99) [reply README.md]: fixed.

**Review by xavier506 — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -4,12 +4,11 @@ Sitio Web Asociación Blockchain de Costa Rica
 ## About Edenia
 
 <div align="center">
-	<a href="https://edenia.com">
-		<img src="https://raw.githubusercontent.com/edenia/.github/master/.github/workflows/images/edenia-logo.png" width="300">
-	</a>
+
+<a href="https://edenia.com"><img width="400" alt="image" src="https://raw.githubusercontent.com/edenia/.github/master/.github/workflows/images/edenia-logo.png"></img></a>
```
Comment (xavier506) [inline README.md:8]: alt="image" should actually be "edenia Logo" 

`alt` tag is a text that describes the image

https://blog.hubspot.com/marketing/image-alt-text

Comment (fvives99) [reply README.md]: fixed

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**